src/host/version.h.in

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <string>
+
+namespace ccf
+{
+  inline static std::string get_ccf_version()
+  {
+    return "@CCF_VERSION@";
```

Comment:
Any reason this is a function rather than just a value (`static constexpr auto ccf_version = "@CCF_VERSION@";`)? The value creates a symbol in the built binary that could be read by other tools.

Reply:
Oh and the other advantage of making this an exported symbol is that we can include the same header in the enclave binaries, and get the same machine-readable symbol there.

Reply:
There was no particular reason for a function and you're right. I'll change that now.